### PR TITLE
My Acclaro - added intro.md

### PR DESCRIPTION
**Pull request by msanz-acclaro**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to COVID delays, unanticipated team leaves, and high volume of connector submissions. 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/My Acclaro/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# My Acclaro
```
Comment (v-eisljami) [inline certified-connectors/My Acclaro/README.md:1]: Readme is not in the correct format. Please refer to this link to see a template of the readme as well as what are the required sections and their limitations. If you have any questions please feel free to contact us.

File: certified-connectors/My Acclaro/intro.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+The Acclaro Web Service API is an extensible set of RESTful interfaces that allow customers and partners to automatically manage translation activities with Acclaro, such as file transfers, order creation, quality ratings and status reporting. With the API in use, source content that resides in commercial or custom developed content management systems can be programmatically sent for translation, then subsequently received and published. Any questions or feedback regarding the Acclaro API can be directed to support@acclaro.com.
```
Comment (v-eisljami) [inline certified-connectors/My Acclaro/intro.md:1]: What is the purpose of this file? There are a lot of overlaps with readme.

File: certified-connectors/My Acclaro/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,13061 @@
...
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete a file from the order",
+        "tags": [
+          "file"
+        ],
+        "description": "Deletes the given file from the order, prior to an order being submitted. If a source file ever needs to be cancelled after it’s been submitted as part of an order, ‘CANCEL FILE’ can be submitted as a comment in /orders/{orderid}/files/{fileid}}/comment. We also request that you separately contact your Acclaro project manager with the source file cancellation request.",
```
Comment (v-eisljami) [inline certified-connectors/My Acclaro/apiDefinition.swagger.json:521]: Please review all descriptions, summaries, titles, etc. They need to be free from grammar and spelling errors.

File: certified-connectors/My Acclaro/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,13061 @@
...
+        }
+      }
+    },
+    "/info/order-types": {
+      "get": {
+        "summary": "Get all supported order types",
+        "tags": [
+          "info"
+        ],
+        "description": "Returns all order types available for the account as an array of Order Types, each order type consisting of a name and a flag “default” which will be zero or one, indicating which order type will be set when none is specified. Note that order types is an optional argument in create order(/orders) and edit order (/orders/{orderId}) intended to assist clients who require this level of order classification for reporting purposes.",
```
Comment (v-eisljami) [inline certified-connectors/My Acclaro/apiDefinition.swagger.json:396]: This description is too long. Please try to rephrase it. You can always check our [string guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance) for more information.

File: certified-connectors/My Acclaro/intro.md (hunk 2 above)
Comment (msanz-acclaro) [reply certified-connectors/My Acclaro/intro.md:1]: this was requested by the ISV certification engineer

File: certified-connectors/My Acclaro/README.md (hunk 1 above)
Comment (msanz-acclaro) [reply certified-connectors/My Acclaro/README.md:1]: this was already approved in my previous PR. I just wanted to add the `intro.md` file into this PR, which was requested as part of the ISV Studio certification (different phase of the certification process)

File: certified-connectors/My Acclaro/apiDefinition.swagger.json (hunk 3 above)
Comment (msanz-acclaro) [reply certified-connectors/My Acclaro/apiDefinition.swagger.json:521]: corrected

File: certified-connectors/My Acclaro/apiDefinition.swagger.json (hunk 4 above)
Comment (msanz-acclaro) [reply certified-connectors/My Acclaro/apiDefinition.swagger.json:396]: fixed

**Comment by msanz-acclaro** [comment]
` Swagger and Change Validation Failing after 51s — Build #20220413.5 failed `

Could I get details on what it is failing? when I import this manually as a custom connector no errors are seen. When I try to go to the details of this error, I get a permission denied.

I have also double checked with swagger and the doc seems to follow all standards, also ran the `paconn` command and no errors are reported.

**Comment by sriyen-msft** [comment]
@msanz-acclaro Please delete the intro.md file from your PR as it is not required to open source it. You can upload it in the activity control area of ISV Studio. Rest everything in the PR looks good.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**